crates/hotshot/src/traits/networking/memory_network.rs

```diff
@@ -147,7 +146,7 @@ impl<M: NetworkMsg, K: SignatureKey> MemoryNetwork<M, K> {
                         Combo::Direct(vec) => {
                             trace!(?vec, "Incoming direct message");
                             // Attempt to decode message
-                            let x = bincode_opts().deserialize(&vec);
+                            let x = Serializer::<0, 1>::deserialize(&vec);
```

Comment:
I think this (and L168) hard-codes the version, though the function isn't explicitly versioned.

I think either `Serializer::CURRENT_STATIC_VERSION`  or `new_0_1` (or similar) just to indicate that the function is explicitly versioned.

Reply:
On re-examination, this is, effectively, a `MEMORY_NET_VERSION_MAJOR`/`MEMORY_NET_VERSION_MINOR` situation...